### PR TITLE
[DOC-3117] CV - SignalFX Connector

### PR DESCRIPTION
**Pull request by n00bitax**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*
* 
Screenshot 2023-07-31 at 7 00 28 PM
Screenshot 2023-07-31 at 7 00 33 PM

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/platform/7_Connectors/static/add-a-git-hub-connector-35a.png
Comment (n00bitax) [inline docs/platform/7_Connectors/static/add-a-git-hub-connector-35a.png:1]: why do we need to replace git hub connector with this image ?

File: docs/platform/7_Connectors/Monitoring-and-Logging-Systems/connect-to-monitoring-and-logging-systems.md [hunk 1]
```diff
@@ -187,31 +187,48 @@ See [Access control](https://cloud.google.com/monitoring/access-control) from
...
+      To create a new secret:
 
-   1. Select **+ New Secret Text**. 
+         1. Select **+ New Secret Text**. 
    
-   2. In the Add new Encrypted Text dialog, select a secret manager, enter a name and value for the secret, an optional description, and a tag, and then select **Save**.  
+         2. In the Add new Encrypted Text dialog, select a secret manager and enter a name for the secret.
    
-   The secret key appears in the **API Token** field.
+         3.  In **Secret Value**, enter the API token generated in Splunk Observability (SignalFx). To learn about creating Splunk Observability (SignalFx) API access token, go to [Create authentication tokens](https://docs.splunk.com/Documentation/Splunk/9.1.0/Security/CreateAuthTokens).
```
Comment (n00bitax) [inline docs/platform/7_Connectors/Monitoring-and-Logging-Systems/connect-to-monitoring-and-logging-systems.md:206]: this is wrong URL , splunk Observability nd splun k entreprise tokens are different
https://docs.splunk.com/Observability/admin/authentication/authentication-tokens/tokens.html

Comment (n00bitax) [reply docs/platform/7_Connectors/Monitoring-and-Logging-Systems/connect-to-monitoring-and-logging-systems.md:206]: we remove an important piece of info :
_We need a token generated from Splunk Observability with Authorization scope_
how will the user now know what scopes need to be assigned to the token?

**Comment by ravilach** [comment]
Lgtm

**Comment by n00bitax** [comment]
lgtm

**Pull request merged**